[Account] Adding warning message to discourage username/password (#25435)
Azure/azure-powershell: src/Accounts/Accounts/ChangeLog.md

-->

## Upcoming Release
* In `Connect-AzAccount` add warning message to deprecate username/password flow

> * In `Get-AzAccessToken` add Breaking Change notification to change AccessToken property from String to Secure String

Reviewer: ```suggestion
* Added a warning message in `Connect-AzAccount` to discourage the use of the username/password (a.k.a ROPC) login flow.
* Preannounced a breaking change in `Get-AzAccessToken` to change `Token` property from `String` to `SecureString`.
``

* Updated `Microsoft.Identity.Client.NativeInterop` to fix the WAM pop window issue in elevated mode [#24967]
* Updated the reference of Azure PowerShell Common to 1.3.98-preview.
* Limited promotional message to interactive scenarios only

> * In `Connect-AzAccount` add warning message to discourage username/password flow

Reviewer 2: ```suggestion
* In `Connect-AzAccount` add warning message to deprecate username/password flow
```

> * In `Get-AzAccessToken` add Breaking Change Notification to Change Access Token from String to Secure String

Reviewer 2: ```suggestion
* In `Get-AzAccessToken` add Breaking Change notification to change AccessToken property from String to Secure String
```

> 

Reviewer 2: Move the change log just under upcoming release
